pygeoapi/templates/jsonld/base.jsonld

```diff
@@ -0,0 +1 @@
+{{ data | to_json | safe }}
```

Comment:
It looks like this file/directory is only used by the tests.  Suggest to remove from `pygeoapi/templates` and store/ref in `tests/data`.